### DISCUSSION TITLE
3.2.1 伪集群启动服务时报错 alert、api、master

### DISCUSSION THREAD

**Discussion by xieyirmvb** [apache/dolphinscheduler#16196] [Q&A]
启动服务时报错

`Field daoPluginConfiguration in org.apache.dolphinscheduler.dao.DaoConfiguration required a bean of type 'org.apache.dolphinscheduler.dao.plugin.api.DaoPluginConfiguration' that could not be found.`

**Comment by liuzhao900928** [comment]
我是在三台服务器龙蜥8.6环境下部署集群模式的时候遇到这个问题了，也是这三个组件启动后就很快down掉，worker-server正常，后面大佬你解决了吗
